Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs

```diff
@@ -715,6 +715,326 @@ public void When_a_deeply_nested_property_of_a_collection_with_an_invalid_value_
+
+            Action act = () => subject.Should().BeEquivalentTo(expected,
+                options => options
+                    .Excluding(x => x.Level.Collection).ThenExcluding(x => x.NextCollection)
+            );
+
+            act.Should().NotThrow();
+        }
+
+        [Fact]
+        public void When_attribute_in_collection_in_collection_is_excluded_with_then_excluding_it_should_not_throw()
+        {
+            // Arrange
+            var subject = new
+            {
+                Text = "Text",
+                Level = new
+                {
+                    Collection = new[]
+                    {
+                        new
+                        {
+                            Number = 1,
+                            NextCollection = new[]
+                            {
+                                new
+                                {
+                                    Text = "Text"
+                                }
+                            }
+                        },
+                        new
+                        {
+                            Number = 2,
+                            NextCollection = new[]
+                            {
+                                new
+                                {
+                                    Text = "Actual"
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            var expected = new
+            {
+                Text = "Text",
+                Level = new
+                {
+                    Collection = new[]
+                    {
+                        new
+                        {
+                            Number = 1,
+                            NextCollection = new[]
+                            {
+                                new
+                                {
+                                    Text = "Text"
+                                }
+                            }
+                        },
+                        new
+                        {
+                            Number = 2,
+                            NextCollection = new[]
+                            {
+                                new
+                                {
+                                    Text = "Expected"
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            Action act = () => subject.Should().BeEquivalentTo(expected,
+                options => options
+                    .Excluding(x => x.Level.Collection).ThenExcluding(x => x.NextCollection).ThenExcluding(x => x.Text)
+            );
+
+            act.Should().NotThrow();
+        }
+
+        [Fact]
+        public void When_attribute_in_collection_is_excluded_with_then_excluding_followed_by_root_level_exclude_it_should_not_throw()
```

Comment:
🔧 Suggestion: "A_nested_exclusion_can_be_followed_by_a_root_level_exclusion"

Reply:
Nice, also changed the following tests name to "A_nested_exclusion_can_be_**preceded**_by_a_root_level_exclusion"